app/controllers/api/sf311_cases_controller.rb

```diff
@@ -16,7 +34,8 @@ def create
 
       render :show
     else
-      render json: { error: 'Error creating new 311 case' }, status: :unprocessable_entity
+      render json: { error: 'Error creating new 311 case' },
```

Comment:
@Mokosha your editor line column limit is pretty restrictive :).

Reply:
80 col. 👍(To be clear this is a preference that works well with many editors, not a restriction.)